uint8 BUTTON=0
uint8 PUSH_RIGHT=1
uint8 PUSH_LEFT=2
uint8 SCOOP_RIGHT=3
uint8 SCOOP_LEFT=4
uint8 SCOOP_CENTER=5
uint8 GRAB=6
uint8 LIFT=7

uint8 command
float64 x
float64 y